json-endpoints/class.wpcom-json-api-list-posts-v1-1-endpoint.php

```diff
@@ -42,7 +42,10 @@ function callback( $path = '', $blog_id = 0 ) {
 		}
 
 		// determine statuses
-		$status = $args['status'];
+		$status = null;
+		if ( isset( $args['status'] ) ) {
+			$status = $args['status'];
+		}
 		$status = ( $status ) ? explode( ',', $status ) : array( 'publish' );
```

Comment:
We can just consolidate this all into this line like 

`$status = ( isset( $args['status'] ) ) ? explode( ',', $args['status'] ) : array( 'publish' );`